pages/watch/[id].tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
+import { Nav } from "../../components";
+import { useAddress } from "@thirdweb-dev/react";
+import { useRouter } from "next/router";
+import { Player, usePlaybackInfo } from "@livepeer/react";
+import axios from "axios";
```

Comment:
any reason to use axios here instead of just using fetch directly? (nitpick)

Reply:
nope, i will replace it with fetch!